MinecraftClient/McClient.cs

```diff
@@ -59,6 +59,7 @@ public class McClient : IMinecraftComHandler
         private object lastKeepAliveLock = new object();
         private int respawnTicks = 0;
         private int gamemode = 0;
+        private int protocolVersion;

```

Comment:
`McClient.cs` should not need to use the protocol version. Protocol-dependant stuff should be set in the protocol handler. Maybe add a OnEntityHealth event in `McClient.cs` and call it from the protocol handler?

Reply:
OK. Give me some time